projects/Mallard/src/authentication/helpers.ts

```diff
@@ -20,4 +24,13 @@ const isStaffMember = (userData: IdentityAuthData) =>
 const canViewEdition = (userData: IdentityAuthData): boolean =>
     userData.membershipData.contentAccess.digitalPack || isStaffMember(userData)
 
-export { canViewEdition, isStaffMember }
+/**
+ * This gets a CAS code for a user if one exists
+ */
+const getCASCode = () =>
+    Promise.all([
+        casCredentialsKeychain.get(),
+        legacyCASUsernameCache.get(),
+    ]).then(([current, legacy]) => (current && current.username) || legacy)
```

Comment:
Can we write this as `current?.username || legacy`?

Reply:
Unfortunately not. We don't have support for that in our project ...yet.